### ISSUE TITLE
Inconsistencies create uncertainty about expected output

### ISSUE THREAD

**Issue by jerodg** [EarnUp/debit-scheduler-takehome#1]
TLDR; Your tests are incorrect or the requirements specification is. Or option C, I'm interpreting the requirements incorrectly.

- If I output data per the readme the tests will fail and conversely if I output data as per the tests I might fail the expected output for the assessment.

- For SAMPLE_RESPONSE_1 the amount is incorrect as the first payment is on the 21st due to the start date and day of the week being the same (Friday). From my understanding of the requirements, the full monthly payment has to be made within the month. As such only 1 payment is made in the first month. 

- SAMPLE_RESPNSE_2 is similar as the next date 2021-05-03 is the same as the start date (Monday) and the next-next date 2021-05-31 is a holiday.

- Given the sample requests, both responses should only have a single debit in the first month.

- The structure of the sample responses differs from the structure of the test responses.
